src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -153,7 +153,7 @@ export class OsdListComponent implements OnInit {
       { prop: 'collectedStates', name: this.i18n('Status'), cellTemplate: this.statusColor },
       { prop: 'stats.numpg', name: this.i18n('PGs') },
       { prop: 'stats.stat_bytes', name: this.i18n('Size'), pipe: this.dimlessBinaryPipe },
-      { name: this.i18n('Usage'), cellTemplate: this.osdUsageTpl },
+      { prop: 'stats.stat_bytes_used', name: this.i18n('Usage'), cellTemplate: this.osdUsageTpl },
```

Comment:
I'm wondering if we really want to sort by `stats.stat_bytes_used` and not by the current percentage.  Under the assumption that the size (`stats.stat_bytes`) of the OSDs is not equal to each other, funny things can happen:

![Screenshot_2019-03-07_13-37-05](https://user-images.githubusercontent.com/8761082/56809579-78748100-6834-11e9-883a-1d74584df871.png)

Reply:
Done